chaincfg/params.go

```diff
@@ -623,6 +651,106 @@ var TestNet3Params = Params{
 	SlpAddressPrefix:    "slptest",
 }
 
+var ChipNetParams = Params{
```

Comment:
If possible it would be nice to link to the docs where you pulled these values.

Reply:
Added reference. fd7bfad07bc70e0fdf51de0da2176e7c68b120e8